deploy/openshift/kiali-configmap.yaml

```diff
@@ -10,3 +10,8 @@ data:
     server:
       port: 20001
       static_content_root_directory: /opt/kiali/console
+    external_services:
+      jaeger:
+        url: https://jaeger-query-istio-system.127.0.0.1.nip.io
```

Comment:
Same "https" here.
I am not sure which one is the good one, at least in my installation of istio 1.0 I exposed the jaeger-query svc in non secure format.